code/obj/item/teleportation.dm

```diff
@@ -192,14 +204,6 @@ Frequency:
 		var/t1 = input(user, "Please select a teleporter to lock in on.", "Target Selection") in L
 		if (user.stat || user.restrained())
 			return
-		var/obj/item/I = user.equipped()
-		if (I != src)
-			if (istype(I, /obj/item/magtractor))
-				var/obj/item/magtractor/mag = I
-				if (mag.holding != src)
-					return
-			else
-				return
```

Comment:
Is there a reason this is missing from your changes? I can't really imagine a situation where this would actually happen, but just in case, maybe?

Reply:
I must have just lost it when I was figuring out the control flow, I'll put it back in.